backend/bracket/models/db/player.py

```diff
@@ -39,7 +41,7 @@ class PlayerMultiBody(BaseModelORM):
 class PlayerToInsert(PlayerBody):
     created: datetime_utc
     tournament_id: TournamentId
-    elo_score: Decimal = Decimal("1200.0")
```

Comment:
I took the liberty here to use the global constant in case the number changes in the future. I wonder if you'd want the start elo to be configurable in the future though.